recipes/kuliya/all/conanfile.py

```diff
@@ -0,0 +1,38 @@
+from conan import ConanFile
+from conan.tools.files import copy, get
+from conan.tools.layout import basic_layout
+import os
+
+required_conan_version = ">=1.50.0"
+
+
+class JsmnConan(ConanFile):
+    name = "kuliya"
+    description = "Algeria's college hierarchy dataset as a C library."
+    license = "MIT"
+    topics = ("kuliya", "dataset", "api", "dz")
+    homepage = "https://dzcode-io.github.io/kuliya/"
```

Comment:
```suggestion
    homepage = "https://github.com/dzcode-io/kuliya"
```

This field's used in some npts that help to autimatically publish new verisons if the homepage points to the repo itself :)